debian/bionic/foreman/rules

```diff
@@ -12,10 +12,11 @@ build:
 	/bin/cp config/database.yml.example config/database.yml
 	/bin/cp config/dynflow/orchestrator.yml.example config/dynflow/orchestrator.yml
 	/bin/cp config/dynflow/worker.yml.example config/dynflow/worker.yml
-	/usr/bin/gem install -f thor -v '1.1.0'
+	/bin/mkdir -p ./vendor/cache && cd ./vendor/cache && /usr/bin/gem fetch thor -v '1.1.0'
```

Comment:
Just wondering: is this something we should patch into a `Gemfile` or some `bundler.d` file instead?

Reply:
It's worse, I can explain on IRC tomorrow